packages/client/src/components/GameScore.tsx

```diff
@@ -1,28 +1,63 @@
-import { Box, Flex, Heading, Text } from 'theme-ui';
-import { GameState, Score } from 'shared';
+import { Box, Button, Flex, Heading, Text } from 'theme-ui';
+import { GameState, GameEvent, Score } from 'shared';
+import { gameIO } from 'lib/socket';
+import { useParams } from 'react-router-dom';
 
 type Props = {
   gameScore: Score[];
   gameState: GameState;
 };
 
+const END_OF_GAME_SCORE = 11;
+
 export const GameScore = ({ gameScore, gameState }: Props) => {
+  const { players } = gameState;
+  const totals = Object.fromEntries(gameScore.map((g, i) => [players[i].username, g.total]));
+
+  const isGameFinished =
+    totals[players[0].username] !== totals[players[1].username]
+      ? totals[players[0].username] >= END_OF_GAME_SCORE || totals[players[1].username] >= END_OF_GAME_SCORE
+      : false;
+
+  const winner =
```

Comment:
Isn't it weird that winner can be a `boolean` or a `string`?

Reply:
do you think it's bad? if it's false it won't render anyway, we can move the check later and always have a winner here but render it only when isGameFinished is true if you prefer